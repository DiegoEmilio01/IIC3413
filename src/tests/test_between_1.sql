SELECT *
FROM T1 as x
WHERE x.b BETWEEN 30 AND 35