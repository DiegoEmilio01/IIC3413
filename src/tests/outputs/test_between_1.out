x.a,x.b
"test-record-0",30
"test-not-record-0",30
"test-record-1",31
"test-record-2",32
"test-record-3",33
"test-record-4",34
"test-record-5",35

 got 7 results.
